### PR TITLE
Update Context class

### PR DESCRIPTION
**Pull request by imvucic**
Issue: KHRGA-56

### CODE REVIEW

File: source/iface/context.rst [hunk 1]
```diff
@@ -18,12 +18,9 @@ Contexts
 
   class context;
```
Comment (PetroBondar) [inline source/iface/context.rst:19]: I believe that we should wait a bit for updates on this [issue](https://github.com/KhronosGroup/SYCL_Reference/issues/72#issuecomment-1750665911), maybe it'll be needed to update the synopsis here

File: source/iface/context.rst [hunk 2]
```diff
@@ -135,9 +136,12 @@ Returns information about the context as determined by ``param``. See
 ::
 
   enum class context : int {
```
Comment (PetroBondar) [inline source/iface/context.rst]: Maybe we should add here something like this:
```
namespace sycl {
namespace info {
<context enum definition>
}
}
```

**Review by PetroBondar — APPROVED** [review]
Generally, all seems good, we still need to wait for some decisions about issues mentioned earlier

File: source/iface/context.rst (hunk 2 above)
Comment (imvucic) [reply source/iface/context.rst]: I was using existing template : 
_enum class context : int {_

Let's proceed like this and wait if Khronos open issue similar https://github.com/KhronosGroup/SYCL_Reference/issues/72

File: source/iface/context.rst (hunk 1 above)
Comment (imvucic) [reply source/iface/context.rst:19]: yes, we will wait response from Mike and update on-flow...

**Review by PetroBondar — APPROVED** [review]

**Pull request closed without merging**